mlflow/server/js/src/experiment-tracking/components/ExperimentRunsTableMultiColumnView2.js

```diff
@@ -179,10 +176,26 @@ export class ExperimentRunsTableMultiColumnView2 extends React.Component {
         },
         {
           headerName: ATTRIBUTE_COLUMN_LABELS.DATE,
-          field: 'startTime',
+          field: 'runDateInfo',
```

Comment:
I am curious on this, Are we getting the same data from a new field now?

Reply:
Sort of. I've added a corresponding new `runDateInfo` property in the data returned from `getRowData`, which is an object aggregating all of the data needed to render the date cell rather than just the start time. And `DateCellRenderer` now accesses these values through its `value` prop instead of the `data` prop that contains data for the whole row. This is needed so that we can correctly implement the equals property to handle deciding whether the cell needs to be re-rendered.